.golangci.yml

```diff
@@ -82,6 +82,7 @@ linters-settings:
           - github.com/wI2L/jsondiff
           - github.com/google/uuid
           - golang.org/x/net/idna
+          - github.com/hashicorp/go-version
```

Comment:
You don't need to this dependency. Just compare the version strings and if they don't match, assume downloading the one from the repo.